libraries/AP_HAL_SITL/CAN.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2015-2016  Intel Corporation. All rights reserved.
```

Comment:
Not only me. If you do a search on this "Intel Corporation" you will find out that most of us work for it.